tests/unit/utils/test_objectives.py

```diff
@@ -56,7 +56,11 @@
 def test_objective_maps_minimizers_to_minimum(
     objective: Callable[[TensorType], TensorType], minimizers: TensorType, minimum: TensorType
 ) -> None:
-    npt.assert_allclose(objective(minimizers), tf.squeeze(minimum), rtol=1e-5)
+    objective_values_at_minisers = objective(minimizers)
```

Comment:
Is minisers an intentional shortening of minimizers? Also, are we generally sticking to US spelling is there no policy on this?

Reply:
That is a typo. I think we are sticking to US.